### ISSUE TITLE
Daily streak shows different values

### ISSUE THREAD

**Issue by ngleb** [adrcotfas/goodtime#278]
If I set "Work day starts at 5:00", then my daily streak counter changes it's value from 5 to 8 and then back to 5.

I couldn't understand how to reproduce this issue. I only noticed it after I removed once short session of 2 minutes (cancelled work session).

**Comment by ngleb** [comment]
Today issue happened again. But it did happen even after set worday start to 00:00.

On February 6, 2022 I did 1 session. Daily streak was showing 2. Workday start is set to 00:00. After I checked statistics and confirmed there was only 1 session on Feb 6, 2022, then checked settings, then daily streak was showing 1 again (i.e. correct value).

**Comment by adrcotfas** [comment]
@ngleb Do you mean February 5? I was unable to reproduce this issue. If the workday start is set to 05:00, the session counter will show all sessions starting from today at 5:00 until tomorrow at 5:00.

**Comment by ngleb** [comment]
No, it was Feb 6 (UTC+7 so it was 3am on Feb 6). Workday start was set to 00:00. I'm not sure how to reproduce it. It just shows one value, then I go to statistics or settings, go back to the main screen - it shows a different value.

![gt](https://user-images.githubusercontent.com/46774/152691649-da646673-e837-482f-98f7-f84aa343b892.png)
